INFO     2023-01-31 11:13:28.466 request id: app - uvicorn.server:serve - Started server process [23768]
INFO     2023-01-31 11:13:28.467 request id: app - uvicorn.server:serve - Started server process [23768]
INFO     2023-01-31 11:13:28.468 request id: app - uvicorn.lifespan.on:startup - Waiting for application startup.
INFO     2023-01-31 11:13:28.469 request id: app - uvicorn.lifespan.on:startup - Waiting for application startup.
INFO     2023-01-31 11:13:28.471 request id: app - uvicorn.lifespan.on:startup - Application startup complete.
INFO     2023-01-31 11:13:28.471 request id: app - uvicorn.lifespan.on:startup - Application startup complete.
INFO     2023-01-31 11:13:32.598 request id: None - main:read_root - Hello MaskAPI!
INFO     2023-01-31 11:13:32.599 request id: app - uvicorn.protocols.http.httptools_impl:send - 127.0.0.1:57454 - "GET / HTTP/1.1" 200
INFO     2023-01-31 11:13:32.873 request id: app - uvicorn.protocols.http.httptools_impl:send - 127.0.0.1:57454 - "GET /favicon.ico HTTP/1.1" 404
INFO     2023-01-31 11:13:37.089 request id: app - uvicorn.protocols.http.httptools_impl:send - 127.0.0.1:57454 - "GET /docs HTTP/1.1" 200
INFO     2023-01-31 11:13:38.232 request id: app - uvicorn.protocols.http.httptools_impl:send - 127.0.0.1:57454 - "GET /openapi.json HTTP/1.1" 200
INFO     2023-01-31 11:13:56.271 request id: None - main:create_upload_files - create_upload_files request
INFO     2023-01-31 11:13:56.271 request id: None - videoService:creatUploadTask - creatUploadTask request self:{'taskUUID': '6530748a2c3e4fd1afa1a0291be62d1d', 'taskFiles': [<starlette.datastructures.UploadFile object at 0x000001E953A633D0>, <starlette.datastructures.UploadFile object at 0x000001E953A63220>], 'uploadImgPath': './img/', 'uploadVideoPath': './video/', 'outputVideoPath': './out_video/'}
INFO     2023-01-31 11:13:56.272 request id: None - videoService:detect_picture - detect_picture filename:QQ图片20230130163116.jpg
INFO     2023-01-31 11:13:56.272 request id: None - videoService:saveFile - saveFile path:./img/,file:<starlette.datastructures.UploadFile object at 0x000001E953A633D0>,ftype:pic
ERROR    2023-01-31 11:13:56.273 request id: None - videoService:saveFile - 不存在该路径 path:./img/
ERROR    2023-01-31 11:13:56.274 request id: None - videoService:creatUploadTask - creatUploadTask saveImgFile code:603
INFO     2023-01-31 11:13:56.278 request id: app - uvicorn.protocols.http.httptools_impl:send - 127.0.0.1:57455 - "POST /tk/greenscreen/ HTTP/1.1" 500
ERROR    2023-01-31 11:13:56.280 request id: app - uvicorn.protocols.http.httptools_impl:run_asgi - Exception in ASGI application

Traceback (most recent call last):

  File "<string>", line 1, in <module>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\multiprocessing\spawn.py", line 116, in spawn_main
    exitcode = _main(fd, parent_sentinel)
               │     │   └ 764
               │     └ 3
               └ <function _main at 0x000001E93A8E73A0>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\multiprocessing\spawn.py", line 129, in _main
    return self._bootstrap(parent_sentinel)
           │    │          └ 764
           │    └ <function BaseProcess._bootstrap at 0x000001E93A8C2940>
           └ <SpawnProcess name='SpawnProcess-1' parent=21744 started>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\multiprocessing\process.py", line 315, in _bootstrap
    self.run()
    │    └ <function BaseProcess.run at 0x000001E93A8ACF70>
    └ <SpawnProcess name='SpawnProcess-1' parent=21744 started>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {'config': <uvicorn.config.Config object at 0x000001E93A8F0F10>, 'target': <bound method Server.run of <uvicorn.server.Server...
    │    │        │    │        └ <SpawnProcess name='SpawnProcess-1' parent=21744 started>
    │    │        │    └ ()
    │    │        └ <SpawnProcess name='SpawnProcess-1' parent=21744 started>
    │    └ <function subprocess_started at 0x000001E952852E50>
    └ <SpawnProcess name='SpawnProcess-1' parent=21744 started>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\uvicorn\_subprocess.py", line 76, in subprocess_started
    target(sockets=sockets)
    │              └ [<socket.socket fd=1544, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 8080)>]
    └ <bound method Server.run of <uvicorn.server.Server object at 0x000001E93A8F0F40>>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\uvicorn\server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ [<socket.socket fd=1544, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 8080)>]
           │       │   │    └ <function Server.serve at 0x000001E9528353A0>
           │       │   └ <uvicorn.server.Server object at 0x000001E93A8F0F40>
           │       └ <function run at 0x000001E951F1E310>
           └ <module 'asyncio' from 'C:\\Users\\42525\\AppData\\Local\\Programs\\Python\\Python39\\lib\\asyncio\\__init__.py'>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x000001E9528A4E40>
           │    └ <function BaseEventLoop.run_until_complete at 0x000001E951F1FD30>
           └ <_WindowsSelectorEventLoop running=True closed=False debug=False>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\asyncio\base_events.py", line 629, in run_until_complete
    self.run_forever()
    │    └ <function BaseEventLoop.run_forever at 0x000001E951F1FCA0>
    └ <_WindowsSelectorEventLoop running=True closed=False debug=False>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
    │    └ <function BaseEventLoop._run_once at 0x000001E951F21820>
    └ <_WindowsSelectorEventLoop running=True closed=False debug=False>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\asyncio\base_events.py", line 1890, in _run_once
    handle._run()
    │      └ <function Handle._run at 0x000001E951E9CD30>
    └ <Handle <TaskWakeupMethWrapper object at 0x000001E953A63100>(<Future finished result=None>)>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
    │    │            │    │           │    └ <member '_args' of 'Handle' objects>
    │    │            │    │           └ <Handle <TaskWakeupMethWrapper object at 0x000001E953A63100>(<Future finished result=None>)>
    │    │            │    └ <member '_callback' of 'Handle' objects>
    │    │            └ <Handle <TaskWakeupMethWrapper object at 0x000001E953A63100>(<Future finished result=None>)>
    │    └ <member '_context' of 'Handle' objects>
    └ <Handle <TaskWakeupMethWrapper object at 0x000001E953A63100>(<Future finished result=None>)>

> File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\uvicorn\protocols\http\httptools_impl.py", line 419, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
                   └ <uvicorn.middleware.proxy_headers.ProxyHeadersMiddleware object at 0x000001E9528C61C0>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\uvicorn\middleware\proxy_headers.py", line 78, in __call__
    return await self.app(scope, receive, send)
                 │    │   │      │        └ <bound method RequestResponseCycle.send of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E953...
                 │    │   │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
                 │    │   └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
                 │    └ <fastapi.applications.FastAPI object at 0x000001E9539E54F0>
                 └ <uvicorn.middleware.proxy_headers.ProxyHeadersMiddleware object at 0x000001E9528C61C0>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\fastapi\applications.py", line 270, in __call__
    await super().__call__(scope, receive, send)
                           │      │        └ <bound method RequestResponseCycle.send of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E953...
                           │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
                           └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\applications.py", line 124, in __call__
    await self.middleware_stack(scope, receive, send)
          │    │                │      │        └ <bound method RequestResponseCycle.send of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E953...
          │    │                │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
          │    │                └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
          │    └ <starlette.middleware.errors.ServerErrorMiddleware object at 0x000001E9539E57F0>
          └ <fastapi.applications.FastAPI object at 0x000001E9539E54F0>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\middleware\errors.py", line 184, in __call__
    raise exc

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\middleware\errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
          │    │   │      │        └ <function ServerErrorMiddleware.__call__.<locals>._send at 0x000001E953A52DC0>
          │    │   │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
          │    │   └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
          │    └ <starlette.middleware.exceptions.ExceptionMiddleware object at 0x000001E9539E57C0>
          └ <starlette.middleware.errors.ServerErrorMiddleware object at 0x000001E9539E57F0>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\middleware\exceptions.py", line 79, in __call__
    raise exc

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\middleware\exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
          │    │   │      │        └ <function ExceptionMiddleware.__call__.<locals>.sender at 0x000001E953A52D30>
          │    │   │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
          │    │   └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
          │    └ <fastapi.middleware.asyncexitstack.AsyncExitStackMiddleware object at 0x000001E9539E5760>
          └ <starlette.middleware.exceptions.ExceptionMiddleware object at 0x000001E9539E57C0>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\fastapi\middleware\asyncexitstack.py", line 21, in __call__
    raise e

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\fastapi\middleware\asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
          │    │   │      │        └ <function ExceptionMiddleware.__call__.<locals>.sender at 0x000001E953A52D30>
          │    │   │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
          │    │   └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
          │    └ <fastapi.routing.APIRouter object at 0x000001E9539E5550>
          └ <fastapi.middleware.asyncexitstack.AsyncExitStackMiddleware object at 0x000001E9539E5760>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\routing.py", line 706, in __call__
    await route.handle(scope, receive, send)
          │     │      │      │        └ <function ExceptionMiddleware.__call__.<locals>.sender at 0x000001E953A52D30>
          │     │      │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
          │     │      └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
          │     └ <function Route.handle at 0x000001E952530790>
          └ APIRoute(path='/tk/greenscreen/', name='create_upload_files', methods=['POST'])

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\routing.py", line 276, in handle
    await self.app(scope, receive, send)
          │    │   │      │        └ <function ExceptionMiddleware.__call__.<locals>.sender at 0x000001E953A52D30>
          │    │   │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
          │    │   └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
          │    └ <function request_response.<locals>.app at 0x000001E9539EA0D0>
          └ APIRoute(path='/tk/greenscreen/', name='create_upload_files', methods=['POST'])

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\routing.py", line 66, in app
    response = await func(request)
                     │    └ <starlette.requests.Request object at 0x000001E953A63820>
                     └ <function get_request_handler.<locals>.app at 0x000001E9539EA160>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\fastapi\routing.py", line 237, in app
    raw_response = await run_endpoint_function(
                         └ <function run_endpoint_function at 0x000001E95253B280>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\fastapi\routing.py", line 163, in run_endpoint_function
    return await dependant.call(**values)
                 │         │      └ {'files': [<starlette.datastructures.UploadFile object at 0x000001E953A633D0>, <starlette.datastructures.UploadFile object at...
                 │         └ <function create_upload_files at 0x000001E9539EA1F0>
                 └ <fastapi.dependencies.models.Dependant object at 0x000001E9539EC8B0>

  File "d:\自己得\maskapi\app\main.py", line 32, in create_upload_files
    reFileName = picToVideo.creatUploadTask(p,request)
                 │          │               │ └ <starlette.requests.Request object at 0x000001E953A63820>
                 │          │               └ <videoService.picToVideo object at 0x000001E9539FB820>
                 │          └ <function picToVideo.creatUploadTask at 0x000001E95258FEE0>
                 └ <class 'videoService.picToVideo'>

  File "d:\自己得\maskapi\app\videoService.py", line 42, in creatUploadTask
    return response.Response(code)
           │        │        └ 603
           │        └ <function Response at 0x000001E95258FCA0>
           └ <module 'response' from 'd:\\自己得\\maskapi\\app\\response.py'>

  File "d:\自己得\maskapi\app\response.py", line 6, in Response
    "msg":Conf.httpcode[err_code]
          │    │        └ 603
          │    └ {}
          └ <class 'conf.Conf'>

KeyError: 603
ERROR    2023-01-31 11:13:56.346 request id: app - uvicorn.protocols.http.httptools_impl:run_asgi - Exception in ASGI application

Traceback (most recent call last):

  File "<string>", line 1, in <module>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\multiprocessing\spawn.py", line 116, in spawn_main
    exitcode = _main(fd, parent_sentinel)
               │     │   └ 764
               │     └ 3
               └ <function _main at 0x000001E93A8E73A0>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\multiprocessing\spawn.py", line 129, in _main
    return self._bootstrap(parent_sentinel)
           │    │          └ 764
           │    └ <function BaseProcess._bootstrap at 0x000001E93A8C2940>
           └ <SpawnProcess name='SpawnProcess-1' parent=21744 started>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\multiprocessing\process.py", line 315, in _bootstrap
    self.run()
    │    └ <function BaseProcess.run at 0x000001E93A8ACF70>
    └ <SpawnProcess name='SpawnProcess-1' parent=21744 started>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {'config': <uvicorn.config.Config object at 0x000001E93A8F0F10>, 'target': <bound method Server.run of <uvicorn.server.Server...
    │    │        │    │        └ <SpawnProcess name='SpawnProcess-1' parent=21744 started>
    │    │        │    └ ()
    │    │        └ <SpawnProcess name='SpawnProcess-1' parent=21744 started>
    │    └ <function subprocess_started at 0x000001E952852E50>
    └ <SpawnProcess name='SpawnProcess-1' parent=21744 started>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\uvicorn\_subprocess.py", line 76, in subprocess_started
    target(sockets=sockets)
    │              └ [<socket.socket fd=1544, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 8080)>]
    └ <bound method Server.run of <uvicorn.server.Server object at 0x000001E93A8F0F40>>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\uvicorn\server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ [<socket.socket fd=1544, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 8080)>]
           │       │   │    └ <function Server.serve at 0x000001E9528353A0>
           │       │   └ <uvicorn.server.Server object at 0x000001E93A8F0F40>
           │       └ <function run at 0x000001E951F1E310>
           └ <module 'asyncio' from 'C:\\Users\\42525\\AppData\\Local\\Programs\\Python\\Python39\\lib\\asyncio\\__init__.py'>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x000001E9528A4E40>
           │    └ <function BaseEventLoop.run_until_complete at 0x000001E951F1FD30>
           └ <_WindowsSelectorEventLoop running=True closed=False debug=False>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\asyncio\base_events.py", line 629, in run_until_complete
    self.run_forever()
    │    └ <function BaseEventLoop.run_forever at 0x000001E951F1FCA0>
    └ <_WindowsSelectorEventLoop running=True closed=False debug=False>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
    │    └ <function BaseEventLoop._run_once at 0x000001E951F21820>
    └ <_WindowsSelectorEventLoop running=True closed=False debug=False>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\asyncio\base_events.py", line 1890, in _run_once
    handle._run()
    │      └ <function Handle._run at 0x000001E951E9CD30>
    └ <Handle <TaskWakeupMethWrapper object at 0x000001E953A63100>(<Future finished result=None>)>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
    │    │            │    │           │    └ <member '_args' of 'Handle' objects>
    │    │            │    │           └ <Handle <TaskWakeupMethWrapper object at 0x000001E953A63100>(<Future finished result=None>)>
    │    │            │    └ <member '_callback' of 'Handle' objects>
    │    │            └ <Handle <TaskWakeupMethWrapper object at 0x000001E953A63100>(<Future finished result=None>)>
    │    └ <member '_context' of 'Handle' objects>
    └ <Handle <TaskWakeupMethWrapper object at 0x000001E953A63100>(<Future finished result=None>)>

> File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\uvicorn\protocols\http\httptools_impl.py", line 419, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
                   └ <uvicorn.middleware.proxy_headers.ProxyHeadersMiddleware object at 0x000001E9528C61C0>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\uvicorn\middleware\proxy_headers.py", line 78, in __call__
    return await self.app(scope, receive, send)
                 │    │   │      │        └ <bound method RequestResponseCycle.send of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E953...
                 │    │   │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
                 │    │   └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
                 │    └ <fastapi.applications.FastAPI object at 0x000001E9539E54F0>
                 └ <uvicorn.middleware.proxy_headers.ProxyHeadersMiddleware object at 0x000001E9528C61C0>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\fastapi\applications.py", line 270, in __call__
    await super().__call__(scope, receive, send)
                           │      │        └ <bound method RequestResponseCycle.send of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E953...
                           │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
                           └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\applications.py", line 124, in __call__
    await self.middleware_stack(scope, receive, send)
          │    │                │      │        └ <bound method RequestResponseCycle.send of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E953...
          │    │                │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
          │    │                └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
          │    └ <starlette.middleware.errors.ServerErrorMiddleware object at 0x000001E9539E57F0>
          └ <fastapi.applications.FastAPI object at 0x000001E9539E54F0>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\middleware\errors.py", line 184, in __call__
    raise exc

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\middleware\errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
          │    │   │      │        └ <function ServerErrorMiddleware.__call__.<locals>._send at 0x000001E953A52DC0>
          │    │   │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
          │    │   └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
          │    └ <starlette.middleware.exceptions.ExceptionMiddleware object at 0x000001E9539E57C0>
          └ <starlette.middleware.errors.ServerErrorMiddleware object at 0x000001E9539E57F0>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\middleware\exceptions.py", line 79, in __call__
    raise exc

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\middleware\exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
          │    │   │      │        └ <function ExceptionMiddleware.__call__.<locals>.sender at 0x000001E953A52D30>
          │    │   │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
          │    │   └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
          │    └ <fastapi.middleware.asyncexitstack.AsyncExitStackMiddleware object at 0x000001E9539E5760>
          └ <starlette.middleware.exceptions.ExceptionMiddleware object at 0x000001E9539E57C0>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\fastapi\middleware\asyncexitstack.py", line 21, in __call__
    raise e

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\fastapi\middleware\asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
          │    │   │      │        └ <function ExceptionMiddleware.__call__.<locals>.sender at 0x000001E953A52D30>
          │    │   │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
          │    │   └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
          │    └ <fastapi.routing.APIRouter object at 0x000001E9539E5550>
          └ <fastapi.middleware.asyncexitstack.AsyncExitStackMiddleware object at 0x000001E9539E5760>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\routing.py", line 706, in __call__
    await route.handle(scope, receive, send)
          │     │      │      │        └ <function ExceptionMiddleware.__call__.<locals>.sender at 0x000001E953A52D30>
          │     │      │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
          │     │      └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
          │     └ <function Route.handle at 0x000001E952530790>
          └ APIRoute(path='/tk/greenscreen/', name='create_upload_files', methods=['POST'])

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\routing.py", line 276, in handle
    await self.app(scope, receive, send)
          │    │   │      │        └ <function ExceptionMiddleware.__call__.<locals>.sender at 0x000001E953A52D30>
          │    │   │      └ <bound method RequestResponseCycle.receive of <uvicorn.protocols.http.httptools_impl.RequestResponseCycle object at 0x000001E...
          │    │   └ {'type': 'http', 'asgi': {'version': '3.0', 'spec_version': '2.3'}, 'http_version': '1.1', 'server': ('127.0.0.1', 8080), 'cl...
          │    └ <function request_response.<locals>.app at 0x000001E9539EA0D0>
          └ APIRoute(path='/tk/greenscreen/', name='create_upload_files', methods=['POST'])

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\starlette\routing.py", line 66, in app
    response = await func(request)
                     │    └ <starlette.requests.Request object at 0x000001E953A63820>
                     └ <function get_request_handler.<locals>.app at 0x000001E9539EA160>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\fastapi\routing.py", line 237, in app
    raw_response = await run_endpoint_function(
                         └ <function run_endpoint_function at 0x000001E95253B280>

  File "C:\Users\42525\AppData\Local\Programs\Python\Python39\lib\site-packages\fastapi\routing.py", line 163, in run_endpoint_function
    return await dependant.call(**values)
                 │         │      └ {'files': [<starlette.datastructures.UploadFile object at 0x000001E953A633D0>, <starlette.datastructures.UploadFile object at...
                 │         └ <function create_upload_files at 0x000001E9539EA1F0>
                 └ <fastapi.dependencies.models.Dependant object at 0x000001E9539EC8B0>

  File "d:\自己得\maskapi\app\main.py", line 32, in create_upload_files
    reFileName = picToVideo.creatUploadTask(p,request)
                 │          │               │ └ <starlette.requests.Request object at 0x000001E953A63820>
                 │          │               └ <videoService.picToVideo object at 0x000001E9539FB820>
                 │          └ <function picToVideo.creatUploadTask at 0x000001E95258FEE0>
                 └ <class 'videoService.picToVideo'>

  File "d:\自己得\maskapi\app\videoService.py", line 42, in creatUploadTask
    return response.Response(code)
           │        │        └ 603
           │        └ <function Response at 0x000001E95258FCA0>
           └ <module 'response' from 'd:\\自己得\\maskapi\\app\\response.py'>

  File "d:\自己得\maskapi\app\response.py", line 6, in Response
    "msg":Conf.httpcode[err_code]
          │    │        └ 603
          │    └ {}
          └ <class 'conf.Conf'>

KeyError: 603
